initial work to use parcel bundler and some other libraries

mrcoles opened the pull request:
This expands on this other PR: https://github.com/eirikb/gifie/pull/4 and uses the parcel bundler to bundle things up (allowing me to import some libs, including a [getUserMedia](https://www.npmjs.com/package/getusermedia) library to make that lookup better supported).

*  Setup with: `yarn install`
*  Run with: `yarn run` (makes it easier to test out locally, also the getUserMedia polyfill seems to do something to get around an https-required warning to use the webcam that I’m getting with the old code)
*  Build with: `yarn build`

It could benefit from moving things into a `src/` directory, but I figured I’d move fewer things around here.

mrcoles committed:
initial work to use parcel bundler and some other libraries

eirikb commented:
Hi.
While I love and use parcel every day, and the project really needs to be freshened up I'm not sure if a bundler is a good idea. Some of the point was that should be so simple a <kbd>ctrl</kbd>+<kbd>u</kbd> would show everything needed. Removing bootstrap and jQuery and going vanilla would be much more appealing than introducing a bundler.
Also I have myself moved away from yarn and back to plain npm, all you introduce here is a yarn lockfile.

mrcoles commented:
OK, yeah, no worries. I was originally futzing around with things to figure out why it wasn’t working and added this to see if it was the getUserMedia call, then identified the fix I put in #4  I figured I’d float the bundler approach by you and only do the extra work to button it up if it was something you’d want for this project.

I’ll close this 📕 

Also, FWIW, I tracked down the source of this error:

> [Deprecation] getUserMedia() no longer works on insecure origins. To use this feature, you should consider switching your application to a secure origin, such as HTTPS. See https://goo.gl/rStTGz for more details.

I was testing locally with `python3 -m http.server 8888` and when I clicked on the link `http://0.0.0.0:8888/` I got the error, but when I switched it to localhost:8888 it worked fine…
